app/src/main/res/layout/medicalhistory_summary_view.xml

```diff
@@ -97,4 +97,29 @@
 
   </com.google.android.material.card.MaterialCardView>
 
+  <com.google.android.material.card.MaterialCardView
+    android:id="@+id/currentSmokerQuestionContainer"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_8"
+    android:visibility="gone"
+    tools:visibility="visible">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="@dimen/spacing_16"
+      android:layout_marginEnd="@dimen/spacing_16"
+      android:orientation="vertical"
+      android:paddingVertical="@dimen/spacing_8">
+
+      <org.simple.clinic.medicalhistory.MedicalHistoryQuestionView
+        android:id="@+id/currentSmokerQuestionView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
```

Comment:
Same comment as before, if this is only child, we can just wrap it in the card view.